exampleSite/config.toml

```diff
@@ -43,6 +43,8 @@ disqusShortname = "yourdiscussshortname"
     custom_css = []
 
 [taxonomies]
+  categories = "categories"
```

Comment:
Are you experiencing any problems with taxonomies or translations?
Because there is no need to define these taxonomies here, they are default in Hugo.
Also, taxonomies are defined like this: `singular = "plural"`.

Reply:
yes, i'm facing 404 page not found,
also translations, it just show " : Development ", i though it should be " tag: Development " or " category: Development "